cms/djangoapps/models/settings/course_details.py

```diff
@@ -45,8 +45,8 @@ def __init__(self, org, course_id, run):
         self.enrollment_end = None
         self.syllabus = None  # a pdf file asset
         self.short_description = ""
-        self.overview = ""  # html to render as the overview
-        self.about_sidebar_html = "" # html to render as the about_sidebar_html
+        self.overview = ''  # html to render as the overview
```

Comment:
@stvstnfrd :  For reverts, do I need to completely revert a previous commit (e3ea86f), and patch it again?  Or, would quickly changing the code back in a new commit accomplish the same thing?  I don't remember if git is tracking my commit history or just the history of the diffs.